src/node_modules/editor/index.js

```diff
@@ -123,7 +136,7 @@ async function render (s) {
 
 let renderTimeout = null
 
-function evaluateSource () {
+function evaluateSource() {
```

Comment:
I use `standardjs`, which prefers `function foobar () {}`. I think this relates to #49.